java/arcs/jvm/host/JvmHost.kt

```diff
@@ -10,18 +10,17 @@
  */
 package arcs.jvm.host
 
+import arcs.core.host.AbstractArcHost
 import arcs.core.host.ArcHost
 import arcs.core.host.ParticleRegistration
-import arcs.core.host.ProdHost
 import arcs.core.util.Time
 import arcs.jvm.util.JvmTime
 
 /**
- * An [ArcHost] that runs isolatable particles that are expected to have no platform
- * dependencies directly on Android APIs.
+ * An [ArcHost] that runs on Java VM platforms.
  */
-open class JvmProdHost(
+open class JvmHost(
```

Comment:
Abstract class?

Reply:
The problem I have is, there is usually a perfectly legitimate implementation in Core that needs to be extended by Android. The version in both core is usable. I could make an AbstractJvmHost, that is extended by JvmHost and AndroidHost, but these seems like overkill, because the JvmHost would be a 1-liner with no overrides.